kubectx

```diff
@@ -32,6 +32,7 @@ USAGE:
   kubectx <NEW_NAME>=<NAME> : rename context <NAME> to <NEW_NAME>
   kubectx <NEW_NAME>=.      : rename current-context to <NEW_NAME>
   kubectx -d <NAME>         : delete context <NAME> ('.' for current-context)
+                              accepts multiple context names separated by a space
```

Comment:
I think the right notation is to do:

     kubectx -d <NAME> [<NAME...>]

then you don't need this line.